vendor/browan/doorwindowsensor.yaml

```diff
@@ -0,0 +1,85 @@
+name: Door/Window Sensor
+description: The Door & Window sensor utilizes LoRaWAN connectivity to communicate the proximity or not of a magnet. The intended use is to place the sensor and magnet on separate elements of a door or window to determine if the door or window is open or closed.The sensor is composed of two parts. The main body contains the active electronics to measure magnetic fields and transmit any changes to a LoRaWAN network. The second part is a permanent magnet of sufficient field strength to be detected by the Hall Effect sensor on the main body.
+
+
+# Hardware versions (optional, use when you have revisions)
+hardwareVersions:
+  - version: '1.0'
+
+# Firmware versions (at least one is mandatory)
+firmwareVersions:
+  - # Firmware version
+    version: '1.0.1'
+    numeric: 1
+    # Corresponding hardware versions (optional)
+    hardwareVersions:
+      - '1.0'
+
+    # LoRaWAN Device Profiles per region
+    # Supported regions are EU863-870, US902-928, AU915-928, AS923,, IN865-867
+    profiles:
+      EU863-870:
+        # Unique identifier of the profile (lowercase, alphanumeric with dashes, max 36 characters)
+        id: doorwindowsensor-profile
+        lorawanCertified: true
+        codec: doorwindowsensor-codec
+      US902-928:
+        id: doorwindowsensor-profile
```

Comment:
Do the EU868 and US915 devices have the same antenna EIRP? This is often a reason to have separate profiles as max in the US is 27 and max in EU is 16.